Paws4Me/Paws4MeTests/loginTest.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Paws4MeTests.swift
+//  Paws4MeTests
+//
+//  Created by Kirsty-Lee Walker on 2022/02/23.
+//
+
+import XCTest
+@testable import Paws4Me
+
+class LoginTest: XCTestCase {
```

Comment:
Don't see any tests for `loginUser(username: String, password: String)`

Reply:
I have not included it. I will include it now as we ideally want to test all code execution paths.